docker-compose.yaml

```
@@ -48,6 +48,11 @@ services:
       internal_network:
         aliases:
           - studio.127.0.0.1.nip.io
+    healthcheck:
+      test: ["CMD", "wget", "--quiet", "--tries=1", "--spider", "http://localhost:8080"]
```

You can do that now, I guess
```suggestion
      test: ["CMD", "wget", "--quiet", "--tries=1", "--spider", "http://localhost:8080/api/are-you-there/"]
```